app/assets/javascripts/application/document_filter.js

```diff
@@ -203,7 +203,7 @@ if(typeof window.GOVUK === 'undefined'){ window.GOVUK = {}; }
       $filterSummary.mustache('documents/_filter_selections', context);
     },
     removeFilters: function(field, removed){
-      var selects = ['topics', 'departments', 'world_locations', 'official_document_status'],
+      var selects = ['topics', 'departments', 'world_locations', 'official_document_status', 'people_ids'],
```

Comment:
Should this just be `people` to be consistent with the others.

Reply:
I think we kept the name `people_ids` because that's what it was originally called here, and to keep it distinct from the people controller.